src/main/java/qupath/AnnotationExchangeExtension/ExportAnnotationServiceJSON.java

```diff
@@ -30,13 +37,38 @@ public ExportAnnotationServiceJSON(QuPathGUI qupath){
         this.qupath = qupath;
     }
 
+    public Collection<Class<? extends PathObject>> getSupportedParentObjectClasses() {
+        List<Class<? extends PathObject>> list = new ArrayList<>(1);
+        list.add(PathAnnotationObject.class);
+        return list;
+    }
+
     public void run(){
         QuPathViewer viewer = qupath.getViewer();
         if (viewer == null || viewer.getServer() == null) {
             logger.error("No Slide Loaded.");
             return;
         }
 
+        PluginRunnerFX runner = new PluginRunnerFX(qupath,false);
```

Comment:
is there an auto formatter for java we can run?

Reply:
I have `SonarLint` installed in my IDE:

https://www.sonarlint.org/intellij/

And I admittedly haven't looked very hard, but I haven't come across an equivalent to `.eslintrc` that I can commit.

I'll see if I have time to look a bit harder so we can standardize the code style.